<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>vProductSetup</class>
 <widget class="QWidget" name="vProductSetup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>950</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1200</width>
    <height>950</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1200</width>
    <height>950</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Arial</family>
    <pointsize>18</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTableWidget" name="tbFinalProduct">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>411</width>
     <height>781</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>14</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="btnPLoad">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>870</y>
     <width>91</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
    </font>
   </property>
   <property name="text">
    <string>Load</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnPNew">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>870</y>
     <width>91</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
    </font>
   </property>
   <property name="text">
    <string>New</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnPSave">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>870</y>
     <width>91</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
    </font>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tbProcess">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>80</y>
     <width>431</width>
     <height>781</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>14</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="btnPDel">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>870</y>
     <width>91</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
    </font>
   </property>
   <property name="text">
    <string>Del</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnP2Load">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>870</y>
     <width>91</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
    </font>
   </property>
   <property name="text">
    <string>Load</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnP2New">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>870</y>
     <width>91</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
    </font>
   </property>
   <property name="text">
    <string>New</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnP2Save">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>870</y>
     <width>91</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
    </font>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnP2Del">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>870</y>
     <width>91</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
    </font>
   </property>
   <property name="text">
    <string>Del</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblPart">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>181</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Products:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblProcess">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>20</y>
     <width>181</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Process:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnP3Load">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>870</y>
     <width>81</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
    </font>
   </property>
   <property name="text">
    <string>Load</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnP3Del">
   <property name="geometry">
    <rect>
     <x>1110</x>
     <y>870</y>
     <width>81</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
    </font>
   </property>
   <property name="text">
    <string>Del</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tbPart">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>80</y>
     <width>321</width>
     <height>781</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="rowCount">
    <number>0</number>
   </property>
   <property name="columnCount">
    <number>0</number>
   </property>
  </widget>
  <widget class="QPushButton" name="btnP3New">
   <property name="geometry">
    <rect>
     <x>950</x>
     <y>870</y>
     <width>81</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
    </font>
   </property>
   <property name="text">
    <string>New</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnP3Save">
   <property name="geometry">
    <rect>
     <x>1030</x>
     <y>870</y>
     <width>81</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
    </font>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblPart_2">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>10</y>
     <width>181</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Parts:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
